src/ambianic/notification.py

```diff
@@ -4,13 +4,52 @@
 import apprise
 import os
 import ambianic
+import pkg_resources
+import yaml
+from requests import post
 
 log = logging.getLogger(__name__)
 
 UI_BASEURL = "https://ui.ambianic.ai"
 
+
+def sendCloudNotification(data):
```

Comment:
@vickywane `sendCloudNotification` is a redundant function. The existing notification class via apprise already allows json notification provider. You just need to pass to it the correct Ambianic Cloud API credentials and URL.
https://github.com/ambianic/ambianic-edge/blob/0291b2abfd76fd9a985a5b09445a505dfd4f83ca/tests/pipeline/test_notify.py#L135

Reply:
One of the requirements of this feature is to grant the least privilege to the edge device by supplying only the credentials needed. 

I am intentionally not utilizing Apprise because I do not want to pass the Mailgun credentials to the Edge device. Currently, the `sendNotification` function simply makes an HTTP request directly to the Cloud API, passing the user's ID and the event detection.  

It is the Cloud API's function to send the email using Mailgun client libraries. 

The existing Apprise configuration would still be valid for users who want to manually configure their device notification.

We have discussed this flow previously and it is also documented in the design diagram attached to this pull request.

Reply:
@ivelin [this](https://github.com/ambianic/ambianic-edge/issues/353) issue has been created as a follow-up to handle the adjustment of the existing `Notification`  class.